scripts/aws-copy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+AMI_ID=$(aws ec2 describe-images --region $SOURCE_LOCATION --filters "Name=name,Values=$IMAGE_NAME" --owners "self" --query 'Images[*].[ImageId]' --output "text")
+
+if [ -z "$AMI_ID" ]
+then
+  echo "Source image not found, exiting"
```

Comment:
Please refer to the variable what you checks to make it clear what's missing.

Reply:
Good idea, fixed